docs/source/contribute/quickstart/index.rst

```diff
@@ -16,7 +16,6 @@ Prerequisite Software
   :includehidden:
 
   build
-  environment
```

Comment:
Can you update this to [a link to the external file instead](https://github.com/NASA-AMMOS/aerie/blob/develop/deployment/Environment.md)? An example of how to do so can be found in `docs/source/contribute/index.rst`, specifically on the last line of the file.

Reply:
Done